GetCertificate/printer.helpers.ts

```diff
@@ -0,0 +1,100 @@
+import {
+  PreferredLanguage,
+  PreferredLanguageEnum
+} from "@pagopa/io-functions-commons/dist/generated/definitions/PreferredLanguage";
+import * as o from "fp-ts/lib/Option";
+import { pipe } from "fp-ts/lib/pipeable";
+import * as moment from "moment";
+import { match } from "ts-pattern";
+import {
+  Certificates,
+  TestCertificate,
+  VacCertificate,
+  VaccinationEntry,
+  RecoveryCertificate
+} from "./certificate";
+
+const DATE_FORMAT_EN = "YYYY-MM-DD";
+const DATE_FORMAT_ITA = "DD-MM-YYYY";
+
+/**
+ * Returns UVCI of the first certificate (either Vaccination, Test or Recovery)
+ *
+ * @param _lang the preferred language (unused)
+ * @param c the Certificate
+ * @returns the UVCI
+ */
+export const printUvci = (
+  _lang: o.Option<PreferredLanguage>,
+  c: Certificates
+): string =>
+  match(c)
+    .when(VacCertificate.is, cv => cv.v[0].ci)
+    .when(TestCertificate.is, ct => ct.t[0].ci)
+    .when(RecoveryCertificate.is, cr => cr.r[0].ci)
+    .exhaustive();
+/**
+ * Returns UVCI of the first certificate (either Vaccination, Test or Recovery)
+ * splitted in two lines
+ *
+ * @param _lang the preferred language (unused)
+ * @param c the Certificate
+ * @returns the UVCI
+ */
+export const formatUvciTwoLines = (uvci: string): string =>
```

Comment:
I think this isn't needed due to markdown wrap text implementantion in app
https://github.com/pagopa/io-app/pull/3127